PollBuddy-Server/frontend/src/pages/GroupJoin/GroupJoin.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import "mdbreact/dist/css/mdb.css";
+import { MDBContainer } from "mdbreact";
+
+export default class GroupJoin extends Component {//this class will likely need to call Groups/new and do more with that...
+  componentDidMount(){
+    this.props.updateTitle("Group Join");
+  }
+
+  constructor() {
+    super();
+    //TODO check if they're logged in
+    console.log(localStorage.getItem("loggedIn"));
+    if (localStorage.getItem("loggedIn") == "true") {
+      console.log("Logged in");
+    } else {
+      console.log("Not logged in");
+      // window.location.replace("/login");
+    }
+    this.state = {
+      groupCode: "",
+      showConfirm: false
+    };
+  }
+
+  handleEnterCode = () => {
+    this.setState({showConfirm: true});
+  }
+
+  handleChange = (e) => {
+    this.setState({groupCode: e.target.value});
+  }
+
+  handleConfirmationResponse(response) {
+    if (response === true) {
+      window.location.replace("/groups/" + this.state.groupCode + "/polls");
+    } else {
+      window.location.replace("/groups");
+    }
+  }
+
+  render() {
+    //TODO check if they're logged in
```

Comment:
Unimportant but just so you know, typical code comment convention is like this: 
```
//This is code that was commented out, usually temporarily
// This is for an information comment. The space is the difference.
// TODO: Is also generally formatted like this
``` 
It definitely doesn't really matter here but I just wanted to point it out as if you go into software development anywhere, that's what you'll see and likely be expected to do. RCOS is all about learning anyways :)